radiography/src/main/java/radiography/DecoratingAppendable.kt

```diff
@@ -0,0 +1,32 @@
+package radiography
+
+internal class DecoratingAppendable(
```

Comment:
"Decorating" is generic, i'd call this "CommaSeparatedAppendable" or something.

Reply:
I'm half tempted to drop Appendable in favor of am interface here, since I only support two of the 3 methods, and really only care about one.

Reply:
Yea, better to define a new type than throw from a standard one.